src/Components/Movie/Movie.js

```diff
@@ -0,0 +1,165 @@
+import "./movie.css";
+import { BigPlayButton, Player } from "video-react";
+import WatchLaterIcon from "@mui/icons-material/WatchLater";
+import ThumbUpIcon from "@mui/icons-material/ThumbUp";
+import ThumbDownIcon from "@mui/icons-material/ThumbDown";
+import DeleteIcon from "@mui/icons-material/Delete";
+import PlaylistAddIcon from "@mui/icons-material/PlaylistAdd";
+import CloseIcon from "@mui/icons-material/Close";
+import Modal from "@mui/material/Modal";
+import Box from "@mui/material/Box";
```

Comment:
suggestion: you can create an index file in the assets folder and export all icons from index file and import them here